cobra_commander.gemspec

```diff
@@ -34,6 +34,7 @@ DESCRIPTION
 
   spec.add_dependency "thor", "~> 0.19.4"
   spec.add_dependency "ruby-graphviz", "~> 1.2.3"
+  spec.add_dependency "oj", "~> 2.18.1"
```

Comment:
Is this really so performance-sensitive that we can't just use the stdlib JSON parser?

Reply:
Maybe.

While this addition does memoize quite a bit, the multiple listings of components (think how many times `nitro_react` will appear in `nitro-web`s tree) means all package.json files will be parsed multiple times. 

I'll leave for the end and do some benchmarking to see the difference.

Reply:
Ok.  You are correct. The difference between `Oj.load` & `JSON.parse` when executing the command against our main repo (with a change to `nitro_react`) is inconsequential. 

<img width="146" alt="screen shot 2017-08-31 at 7 22 03 am" src="https://user-images.githubusercontent.com/12770108/29920926-b92b4bd6-8e1d-11e7-9590-00848eebde51.png">

<img width="128" alt="screen shot 2017-08-31 at 7 22 17 am" src="https://user-images.githubusercontent.com/12770108/29920927-bb3b905c-8e1d-11e7-9523-df0b30cc8817.png">

But why does this command take so long in ci then?

<img width="1113" alt="screen shot 2017-08-31 at 7 22 54 am" src="https://user-images.githubusercontent.com/12770108/29920941-cf59cafe-8e1d-11e7-8171-226554213883.png">

Reply:
I'm not sure why that's so slow, but I doubt it's because of JSON parsing.